src/test/kotlin/uk/gov/dluhc/printapi/testsupport/testdata/model/PrintResponseBuilder.kt

```diff
@@ -0,0 +1,41 @@
+package uk.gov.dluhc.printapi.testsupport.testdata.model
+
+import uk.gov.dluhc.printapi.printprovider.models.BatchResponse
+import uk.gov.dluhc.printapi.printprovider.models.PrintResponse
+import uk.gov.dluhc.printapi.printprovider.models.PrintResponses
+import uk.gov.dluhc.printapi.testsupport.testdata.DataFaker
+import java.util.Date
+import java.util.UUID
+import kotlin.random.Random
+
+fun buildPrintResponses(
+    batchResponses: List<BatchResponse> = listOf(buildBatchResponse()),
+    printResponses: List<PrintResponse> = listOf(buildPrintResponse()),
+): PrintResponses =
+    PrintResponses()
+        .withBatchResponses(batchResponses)
+        .withPrintResponses(printResponses)
+
+fun buildBatchResponse(
+    batchId: String = Random.nextInt().toString(),
```

Comment:
You can use `aValidBatchId()` here - it better reflects what a batch ID looks like 👍